orttraining/orttraining/test/python/orttraining_test_ortmodule_deepspeed_zero_stage_1.py

```diff
@@ -167,19 +168,28 @@ def main():
         device = "cpu"
 
     ## Data loader
+    data_dir = '/mnist'
```

Comment:
Same comment as before. Default values cannot be hard-coded folders that only exist in CI.

Reply:
Although, these values are hard coded, they are not default values since they are overwritten in case path does not exist. Nevertheless, I have updated the script such that the path to the dataset is passed from the yml file.